eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
```

Comment:
We should have a goal of moving this to eng/common at some point after we get everything working.